test/filter.js

```diff
@@ -152,6 +152,24 @@ test('With an esri style envelope', t => {
   run('trees', options, 29744, t)
 })
 
+test('With an esri style envelope and a missing', t => {
```

Comment:
missing the word "geometry" at the end of the test name?

Reply:
oops, thanks